libmamba/src/api/create.cpp

```diff
@@ -45,7 +45,22 @@ namespace mamba
                 if (ctx.prefix_params.target_prefix == ctx.prefix_params.root_prefix)
                 {
                     LOG_ERROR << "Overwriting root prefix is not permitted";
-                    throw std::runtime_error("Aborting.");
+                    throw mamba_error(
+                        "Overwriting root prefix is not permitted - aborting.",
```

Comment:
I would prefer if this message was in both cases (the aborting is useful in the log too), so sometihing like:
```c++
const auto message = "Overwriting root prefix is not permitted - aborting.";
LOG_ERROR << message;
throw mamba_error(message, mamba_error_code::incorrect_usage);
```

same for the other one

Reply:
Of course, addressed in a rebased commit.